core/controllers/tasks_test.py

```diff
@@ -0,0 +1,333 @@
+                thread_id, self.user_id_b, None, None, 'user b message')
+
+            #check that there are two messages in thread.
+            messages = feedback_services.get_messages(thread_id)
+            self.assertEqual(len(messages), 2)
+
+            #telling tasks.py to send email to User 'A'.
+            payload = {
+                'user_id': self.user_id_a}
+            taskqueue_services.enqueue_email_task(
+                feconf.TASK_URL_FEEDBACK_MESSAGE_EMAILS, payload, 0)
+
+            #check that there are no feedback emails sent to User 'A'.
+            messages = self.mail_stub.get_sent_messages(to=self.USER_A_EMAIL)
+            self.assertEqual(len(messages), 0)
+
+            #send task and subsequent email to User 'A'.
+            self.process_and_flush_pending_tasks()
+            messages = self.mail_stub.get_sent_messages(to=self.USER_A_EMAIL)
+            expected_message = (
+                'Hi userA,\n\nNew update to thread "a subject"'
+                ' on Title:\n- userB: user b message\n(You received'
+                ' this message because you are a participant in this thread.)'
+                '\n\nBest wishes,\nThe Oppia team\n\nYou can change your email'
+                ' preferences via the Preferences page.')
+
+            #assert that the message is correct.
+            self.assertEqual(len(messages), 1)
+            self.assertEqual(messages[0].body.decode(), expected_message)
+
+            #create another message that is len = 201.
+            user_b_message = 'B' * 201
+            feedback_services.create_message(
+                thread_id, self.user_id_b, None, None, user_b_message)
+
+            #check that there are three messages in thread.
+            messages = feedback_services.get_messages(thread_id)
+            self.assertEqual(len(messages), 3)
+
+            #telling tasks.py to send email to User 'A'.
+            payload = {
+                'user_id': self.user_id_a}
+            taskqueue_services.enqueue_email_task(
+                feconf.TASK_URL_FEEDBACK_MESSAGE_EMAILS, payload, 0)
+
+            #check that there is one feedback email sent to User 'A'.
+            messages = self.mail_stub.get_sent_messages(to=self.USER_A_EMAIL)
+            self.assertEqual(len(messages), 1)
+
+            #send task and subsequent email to User 'A'.
+            self.process_and_flush_pending_tasks()
+            messages = self.mail_stub.get_sent_messages(to=self.USER_A_EMAIL)
+            expected_message = (
+                'Hi userA,\n\nNew update to thread "a subject"'
+                ' on Title:\n- userB:' + 'B' * 200 + '...' + ' e\n(You received'
+                ' this message because you are a participant in this thread.)'
+                '\n\nBest wishes,\nThe Oppia team\n\nYou can change your email'
+                ' preferences via the Preferences page.')
+
+            #check that greater than 200 word message is correct.
```

Comment:
I was just wondering, do we enforce a space after the # for comments? Seems like the codebase follows it, but lint checks pass here. @apb7 or @seanlip could you confirm?

Reply:
Also, shouldn't these start with a capital letter? /cc @apb7

Reply:
Done.

Reply:
@seanlip: I don't think we have a check to enforce a single space after `#` in a comment. @lilithxxx, can you please go ahead and file an issue to add this check? This can be added in `_check_comments` function of the pre-commit script. Thanks!

Reply:
@apb7 opened #5843